container/load.bzl

```diff
@@ -33,9 +33,12 @@ container_import(
     layers = glob(["*.tar.gz"]),
 )""")
 
-    loader = repository_ctx.attr._loader_linux
+    loader = repository_ctx.attr._loader_linux_amd64
     if repository_ctx.os.name.lower().startswith("mac os"):
         loader = repository_ctx.attr._loader_darwin
+    arch = repository_ctx.execute(["uname", "-m"]).stdout.strip()
```

Comment:
Unfortunately this will most likely break on Windows. Also given we don't have a Windows CI, it'll be hard to validate either way.

I think I want to stay away from having to maintain platform specific ways of detecting the architecture. Instead could you add a new attribute that defaults to amd64 but could be explicitly specified to be s390x?

Reply:
@smukherj1 Please review updated `load.bzl` and `pull.bzl` files.